201906_highres_foregrounds_extragalactic_tophat/README.md

```diff
@@ -0,0 +1,114 @@
+| MFF1 |  93 |
+| MFF2 |  145 |
+| MFS1 |  93 |
+| MFS2 |  145 |
+| UHF1 |  225 |
+| UHF2 |  280 |
+
+Beams are now read from `sotodlib`.
+
+**Important notice about beams**: The beams used for all previous releases were read from `SO_Noise_Calculator_Public_20180822.py` and had obsolete values for SAT (in arcmin):
+
+```
+beam_SAT_27 = 91.
+beam_SAT_39 = 63.
+beam_SAT_93 = 30.
+beam_SAT_145 = 17.
+beam_SAT_225 = 11.
+beam_SAT_280 = 9.
+```
+
+The new SAT beams are instead:
+
+```
+beam_SAT_27 = 144.
+beam_SAT_39 = 99.
+beam_SAT_93 = 42.
+beam_SAT_145 = 27.
+beam_SAT_225 = 19.
+beam_SAT_280 = 17.
+```
+
+LAT beams are unchanged.
+
+## Available maps
+
+HEALPix maps at high resolution (nside 4096) and low resolution (nside 512), these models are deterministic, so we have
+a set for each resolution for all channels. All maps are full-sky, see [the `201906_noise_no_lowell`  release for hitmaps](https://github.com/simonsobs/map_based_simulations/tree/master/201906_noise_no_lowell).
+
+Reference frame for the maps is **Equatorial**.
+The `ell_max` for the harmonics transform is `3*Nside-1`.
+
+**Location at NERSC**:
+
+    /project/projectdirs/sobs/v4_sims/mbs/201906_highres_foregrounds_extragalactic_tophat
+
+The naming convention is:
+
+    {nside}/{content}/{num:04d}/simonsobs_{content}_uKCMB_{telescope}{band}_nside{nside}_{num:04d}.fits"
+
+where:
+
+* `content` is in `[dust, synchtrotron, freefree, ame]` for galactic components and `[ksz, tsz, cib]` for extragalactic components
+* `content` for the available CMB is `[cmb, cmb_unlensed, cmb_lensed_solardipole]`, it is the same CMB realization, see the [WebSky documentation for details](https://so-pysm-models.readthedocs.io/en/latest/models.html#websky), `cmb` is the lensed version and `cmb_lensed_solardipole` is equal to `cmb` with the dipole component replaced by the [HFI 2018 solar dipole](https://wiki.cosmos.esa.int/planck-legacy-archive/index.php/Map-making#HFI_2018_Solar_dipole), this was
+  [requested by @keskitalo to highligh temperature to polarization leakage in LAT](https://github.com/simonsobs/so_pysm_models/issues/32#issuecomment-504481401). 
+* CIB map for LF1 is missing, see [issue on Github](https://github.com/simonsobs/map_based_simulations/issues/17)
+* `num` is `0`
+* `telescope` is `sa` or `la`
+* `band` is the channel band, the options are: 
+
+[TODO] backup to tape. Backed up to tape in `~zonca/sobs/mbs/201906_highres_foregrounds_extragalactic_tophat`.
+
+## Combined maps
+
+Also created a single set of maps which is the sum of all components (for CMB `cmb_lensed_solardipole`) to be used as input to some of the TOD simulations (which don't require running PySM on the fly).
+They are in the same folder and same naming convention with `content` equal to `combined_solardip`, e.g.:
+
+    /global/project/projectdirs/sobs/v4_sims/mbs/201906_highres_foregrounds_extragalactic_tophat/512/combined_solardip/0000/simonsobs_combined_solardip_uKCMB_laMFF1_nside512_0000.fits
+
```

Comment:
@keskitalo see the combined maps here,

for the pipeline you need:
* see also this commit in `sotodlib` https://github.com/simonsobs/sotodlib/pull/23/commits/923924cd53ca1672817fc58e1d8d1e6691e03029
* update `so_pysm_models` to the latest master
* update `pysm` to the latest master